Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -417,11 +409,11 @@ final class AppDelegateViewModelTests: TestCase {
     XCTAssertEqual(["App Open", "Opened App"], trackingClient.events)
 
     self.vm.inputs.applicationDidEnterBackground()
-    XCTAssertEqual(["App Open", "Opened App", "App Close", "Closed App"], trackingClient.events)
```

Comment:
Decided to not track `App Close` anymore?

Reply:
Yep, that's right.